charts/seed-controlplane/charts/kube-apiserver/values.yaml

```
@@ -181,3 +181,6 @@ mountHostCADirectories:
 
 #maxNonMutatingRequestsInflight: 400
 #maxMutatingRequestsInflight: 200
+
+reversedVPN:
+  enabled: true
```

```suggestion
  enabled: false
```

Maybe the default should be `false` here as it's only alpha.